src/OnboardingSPA/pages/Steps/Complete/contents.js

```diff
@@ -0,0 +1,44 @@
+import { __, sprintf } from '@wordpress/i18n';
+
+import { translations } from '../../../utils/locales/translations';
+
+const getContents = ( brandName ) => {
+	return {
+		loader: {
+			title: sprintf(
+				/* translators: 1: Brand 2: site */
+				__(
+					'Hang tight, we’re building your %s %s',
+					'wp-module-onboarding'
+				),
+				brandName,
+				translations( 'site' )
+			),
+			subtitle: __(
+				'We’re assembling your unique design and installing useful tools',
+				'wp-module-onboarding'
+			),
+		},
+		errorState: {
+			title: sprintf(
+				/* translators: 1: Brand 2: site */
+				__(
+					'Hang tight, we’re building your %s %s',
```

Comment:
Maybe we can ask David about these error page messages? As they are quite misleading where as this is a still page and nothing is done in background.

Reply:
It does say an error occurred below the "installing/building" so I believe we should be fine with this and if I am not wrong this came from the Bluerock pattern which also shows something similar to this. But will bring it up.